src/SFA.DAS.ApprenticeCommitments.Database/Tables/CommitmentStatement.sql

```diff
@@ -21,13 +20,10 @@ CREATE TABLE [dbo].[CommitmentStatement]
     [HowApprenticeshipDeliveredCorrect] BIT NULL, 
     [ApprenticeshipConfirmed] bit NULL,
     CONSTRAINT PK_CommitmentStatement_Id PRIMARY KEY CLUSTERED ([Id]),
-    CONSTRAINT PK_CommitmentStatement_ApprenticeshipId_CommitmentsApprenticeshipId_ApprovedOn UNIQUE ([ApprenticeshipId], [CommitmentsApprenticeshipId], [ApprovedOn]),
-	CONSTRAINT FK_CommitmentStatement_ApprenticeId FOREIGN KEY ([ApprenticeId]) REFERENCES [dbo].[Apprentice] ([Id])
+    CONSTRAINT FK_CommitmentStatement_Apprenticeship_ApprenticeshipId FOREIGN KEY ([ApprenticeshipId]) REFERENCES [Apprenticeship] ([Id])
```

Comment:
tiny comments:
I think the name should be 
FK_CommitmentStatement_ApprenticeshipId.

also not sure  why ApprovedOn has a default value (it's date comes from Approvals)

Why not call it 'EmployerProviderApprovedOn'

Reply:
I have renamed the constraint - the original name was the one EF came up with, I don't think it really matters.

I have renamed the date to `CommitmentsApprovedOn` as that matches the existing `CommitmentsApprenticeshipId` that also comes from Commitments.  I have also removed the default date - that was an attempt to not require existing tables to be dropped, but I think we're past that now! 😄